20_beta_diversity.Rmd

```diff
@@ -498,16 +498,60 @@ different community composition.
 This method is implemented in the `vegan` package in the function
 [`adonis2`](https://www.rdocumentation.org/packages/vegan/versions/2.4-2/topics/adonis).
 
-We can get equal result by first performing distance-based redundancy analysis (dbRDA),
-and then applying permutational test for result of redundancy analysis. Advantage is 
-that by doing so we can get coefficients: how much each taxa affect to the result.
+**Note:**
 
+It is recommended to `by = "margin"`. It specifies that each variable's marginal
+effect is analyzed individually. 
+
+When `by = "terms"`  the order of variables matters;
+each variable is analyzed sequentially, and the result is different when more than 1 variable is
+introduced and their order is differs.
 
 ```{r}
 if( !require(vegan) ){
     BiocManager::install("vegan")
     library("vegan")
 }
+data("enterotype")
+enterotype <- transformSamples(enterotype, method = "relabundance")
+# Drop those samples that do not have meta dtaa
+enterotype <- 
+    enterotype[ , !rowSums(is.na(colData(enterotype)[, c("Nationality", "Gender","ClinicalStatus")]) > 0 ) ]
+
+# Multiple variables, by = "margin"
+set.seed(75)
+adonis2(t(assay(enterotype ,"relabundance")) ~ Nationality + Gender + ClinicalStatus,
+        by = "margin",
+        data = colData(enterotype),
+        permutations = 99)
+
+# Multiple variables, by = "margin"
+set.seed(75)
+adonis2(t(assay(enterotype ,"relabundance")) ~ ClinicalStatus + Nationality + Gender,
+        by = "margin",
+        data = colData(enterotype ),
+        permutations = 99)
+
+
+# Multiple variables, default: by = "terms"
+set.seed(75)
+adonis2(t(assay(enterotype ,"relabundance"))  ~ Nationality + Gender + ClinicalStatus,
+        data = colData(enterotype ),
+        permutations = 99)
+
+# Multiple variables, default: by = "terms"
+set.seed(75)
+adonis2(t(assay(enterotype ,"relabundance"))  ~ ClinicalStatus + Nationality + Gender,
+        data = colData(enterotype),
+        permutations = 99)
```

Comment:
This is informative but I am wondering if it is a bit much, to keep OMA compact? One option would be to add here "Task" suggesting that the user implements & tries this out themselves (just mention function names and changing variable order compared to the first example). Or this comparison could be on additional supplementary page and here just a link.